e2e-tests/batch-mode-tests.sh

```diff
@@ -13,66 +12,31 @@ function print_message() {
+#     Directory contains parquet tools jar
+#     as well as subdirectory of parquet file output
+#   Subdirectory to search for parquet files. 
+#      NON_JDBC or JDBC
+#   Optional: Flag to specify whether test is being 
+#      done on docker network.
 #######################################
 function setup() {
-  if [[ -z "${ROOT_PATH}" ]]; then
-    echo "ERROR: environment variable ROOT_PATH not set!"
-    exit 1
+  HOME_PATH=$1
+  rm -rf ${HOME_PATH}/fhir
+  rm -rf ${HOME_PATH}/$2/*.json
+  
+  OPENMRS_URL='http://localhost:8099'
+  SINK_SERVER='http://localhost:8098'
+
+  if [[ $3 = "--use_docker_network" ]]; then
+    OPENMRS_URL='http://openmrs:8080'
+    SINK_SERVER='http://sink-server:8080'
   fi
-  cd ${ROOT_PATH}/pipelines
-  HOME_PATH=$(pwd)
-
-  print_message "BUILDING THE ANALYTICS PROJECT"
-  mvn compile
-
-  print_message "STARTING SERVERs"
-  docker-compose -f ${ROOT_PATH}/docker/openmrs-compose.yaml up -d --remove-orphans
-  openmrs_start_wait_time=0
-  contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 \
-     --connect-timeout 5 --max-time 20 http://localhost:8099/openmrs/ws/fhir2/R4/Patient \
-     2>/dev/null | cut -d ";" -f 1)
-  until [[ ${contenttype} == "application/fhir+json" ]]; do
-    sleep 60s
-    print_message "WAITING FOR OPENMRS SERVER TO START"
-    contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 \
-      --connect-timeout 5 --max-time 20 http://localhost:8099/openmrs/ws/fhir2/R4/Patient \
-      2>/dev/null | cut -d ";" -f 1)
-    ((openmrs_start_wait_time += 1))
-    if [[ ${openmrs_start_wait_time} == 20 ]]; then
-      print_message "TERMINATING TEST AS OPENMRS TOOK TOO LONG TO START"
-      exit 1
-    fi
-  done
-  print_message "OPENMRS SERVER STARTED SUCCESSFULLY"
-
-  docker-compose -f ${ROOT_PATH}/docker/sink-compose.yml up -d
-  fhir_server_start_wait_time=0
-  fhir_server_status_code=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET \
-    -u hapi:hapi --connect-timeout 5 --max-time 20 \
-    http://localhost:8098/fhir/Observation 2>/dev/null)
-  until [[ ${fhir_server_status_code} -eq 200 ]]; do
-    sleep 1s
-    print_message "WAITING FOR FHIR SERVER TO START"
-    fhir_server_status_code=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET \
-      -u hapi:hapi --connect-timeout 5 --max-time 20 \
-      http://localhost:8098/fhir/Observation 2>/dev/null)
-    ((fhir_server_start_wait_time += 1))
-    if [[ fhir_server_start_wait_time == 10 ]]; then
-      print_message "TERMINATING AS FHIR SERVER TOOK TOO LONG TO START"
-      exit 1
-    fi
-  done
-  print_message "FHIR SERVER STARTED SUCCESSFULLY"
-
-  TEST_DIR_FHIR=$(mktemp -d -t analytics_tests__XXXXXX_FHIRSEARCH)
-  TEST_DIR_JDBC=$(mktemp -d -t analytics_tests__XXXXXX_JDBC)
-  JDBC_SETTINGS="--jdbcModeEnabled=true"
 }
 
-
 #######################################
 # Function to use count resources in openmrs server
 # Globals:
```

Comment:
Please update `Globals:` as this now relies on more variables; please also distinguish between read and write.